### ISSUE TITLE
 Cannot destructure property 'locales' of 'siteData.themeConfig' as it is undefined.

### ISSUE THREAD

**Issue by YangQuan666** [vuejs/vitepress#379]
### Describe the bug

After upgrading the vitepress version to `0.17.0`, The following error is prompted：
**TypeError: Cannot destructure property 'locales' of 'siteData.themeConfig' as it is undefined**

### Reproduction

1.  Change the version of `vitepress` in the `package.json` file from "^0.16.1" to "^0.17.0",
2. run `yarn install`
3. then run `yarn run docs:dev`
4. which prompts the following error：
```bash
Debugger attached.
yarn run v1.22.11
$ vitepress dev docs
Debugger attached.
vitepress v0.17.0
vite v2.5.2
failed to start server. error:
 TypeError: Cannot destructure property 'locales' of 'siteData.themeConfig' as it is undefined.
    at Object.createLangDictionary (C:\Users\yangquan6\Documents\yangquan666.github.io\node_modules\vitepress\dist\node\shared.js:27:13)
    at resolveSiteData (C:\Users\yangquan6\Documents\yangquan666.github.io\node_modules\vitepress\dist\node\config.js:80:25)
    at Object.resolveConfig (C:\Users\yangquan6\Documents\yangquan666.github.io\node_modules\vitepress\dist\node\config.js:25:24)
    at async Object.createServer (C:\Users\yangquan6\Documents\yangquan666.github.io\node_modules\vitepress\dist\node\server.js:8:20)
Waiting for the debugger to disconnect...
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Waiting for the debugger to disconnect...
```

### Expected behavior

The project starts normally and does not report errors

### System Info

```shell
System:
    OS: Windows 10 10.0.19042
    CPU: (6) x64 Intel(R) Core(TM) i5-9400 CPU @ 2.90GHz
    Memory: 8.29 GB / 15.87 GB
  Binaries:
    Node: 14.17.6 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.11 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 6.14.15 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.19041.1023.0), Chromium (92.0.902.84)
    Internet Explorer: 11.0.19041.1
```

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/coc)
- [X] Read the [docs](https://vitepress.vuejs.org/).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by reactjser** [comment]
Same issue, have you resolved this?

**Comment by YangQuan666** [comment]
> Same issue, have you resolved this?

Not solved yet, it looks like a bug🤔

**Comment by johanjvrens** [comment]
Same issue on mac with npm

**Comment by posva** [comment]
Temporary fix is to add a `themeConfig` entry in your config file:

```js
// docs/.vitepress/config.js

module.exports = {
  themeConfig: {},
}
```

**Comment by ydcjeff** [comment]
Fixed by 034c7375ad2de4b42c0ac861c2dd18183511771d
